tests/serverpod_test_server/test_e2e/method_streaming/open_stream_test.dart

```diff
@@ -0,0 +1,21 @@
+@Timeout(Duration(minutes: 5))
+
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test(
+      'Given a client '
+      'when calling an endpoint multiple times in quick succession '
+      'then all streams complete successfully', () async {
+    var client = Client(serverUrl);
+    var numMessages = 1000;
+    var stream;
+    for (var i = 0; i < 10; i++) {
+      stream = client.methodStreaming.intStreamFromValue(numMessages);
+    }
+
+    await expectLater(stream.last, completes);
+  });
+}
```

Comment:
Suggestion: Also add a test case where a bunch of streams from the same endpoint method are opened, and then read, paused, and resumed, in order to check pause-resume behavior with multiple streams open. (Unless that exists elsewhere already?)

Reply:
Good suggestion, added! 👍